sql/core/src/main/resources/org/apache/spark/sql/execution/ui/static/spark-sql-viz.js

```diff
@@ -70,6 +71,10 @@ function setupTooltipForSparkPlanNode(nodeId) {
     })
 }
 
+// labelSeparator should be a non-graphical character in order not to affect the width of boxes.
+var labelSeparator = "\x01";
+var stageAndTaskMetricsPattern = "^(.*)(\\(stage.*attempt.*task[^)]*\\))(.*)$";
```

Comment:
Not related to this PR, but a minor comment: now we display task info like `stage 1 (attempt 2): task 3`. A more standard (and simpler) way to refer to a stage is `stage 1.2: task 3`.

@sarutak can you help fix it?

Reply:
That makes sense. I'll fix it.